### PR TITLE
Fix missing wordpress.php configuration file

### PR DESCRIPTION
**Pull request by ogorzalka**
Hi @jlambe 

This patch fixes the #832 issue :)

Have a good day !

### CODE REVIEW

File: src/Core/Bootstrap/ConfigurationLoader.php [hunk 1]
```diff
@@ -79,6 +74,21 @@ protected function loadConfigurationFiles(Application $app, RepositoryContract $
...
 
+    /**
+     * Get the path to the routes cache file.
+     *
+     * @return string
+     */
+    protected function maybeForceWpConfigInclude()
+    {
+        // Avoid duplicate constants definitions.
+        if (defined('AUTH_KEY')) {
```
Comment (jlambe) [inline src/Core/Bootstrap/ConfigurationLoader.php:85]: @ogorzalka Could you move this condition into its own method and replace it here as well as inside the `loadConfigurationFiles` method please ?

**Review by jlambe — CHANGES_REQUESTED** [review]
@ogorzalka Thanks for these updates. Just one minor change before I can merge it.

**Comment by ogorzalka** [comment]
@jlambe updated ;)

File: src/Core/Bootstrap/ConfigurationLoader.php [hunk 2]
```diff
@@ -63,12 +58,12 @@ public function bootstrap(Application $app)
...
+
         $files = $this->getConfigurationFiles($app);
 
         if (! isset($files['app'])) {
             throw new Exception('Unable to load the "app" configuration file.');
         }
-
         foreach ($files as $key => $path) {
             // Avoid duplicate constant definitions.
             if ('wordpress' === $key && defined('AUTH_KEY')) {
```
Comment (jlambe) [inline src/Core/Bootstrap/ConfigurationLoader.php:69]:
```suggestion
            if ('wordpress' === $key && $this->isWordPressConfigLoaded()) {
```

File: src/Core/Bootstrap/ConfigurationLoader.php [hunk 3]
```diff
@@ -79,6 +74,30 @@ protected function loadConfigurationFiles(Application $app, RepositoryContract $
...
+        }
+        $cacheConfig = app()->getCachedConfigPath('config.php');
+        file_exists($cacheConfig) ? require_once app()->configPath('wordpress.php') : null;
+    }
+
+    /**
+     * Check if the WordPress config constants are already defined.
+     * @return bool
+     */
+    protected function isWordPressConfigAlreadyLoaded(): bool
```
Comment (jlambe) [inline src/Core/Bootstrap/ConfigurationLoader.php:96]:
```suggestion
    protected function isWordPressConfigLoaded(): bool
```

File: src/Core/Bootstrap/ConfigurationLoader.php [hunk 4]
```diff
@@ -79,6 +74,30 @@ protected function loadConfigurationFiles(Application $app, RepositoryContract $
         }
     }
 
+    /**
+     * Get the path to the routes cache file.
+     *
+     * @return string
```
Comment (jlambe) [inline src/Core/Bootstrap/ConfigurationLoader.php:80]: Remove docblock return statement and add `:void` return statement to method signature.

File: src/Core/Bootstrap/ConfigurationLoader.php [hunk 5]
```diff
@@ -79,6 +74,30 @@ protected function loadConfigurationFiles(Application $app, RepositoryContract $
...
+     */
+    protected function maybeForceWpConfigInclude()
+    {
+        // Avoid duplicate constants definitions.
+        if ($this->isWordPressConfigAlreadyLoaded()) {
+            return;
+        }
+        $cacheConfig = app()->getCachedConfigPath('config.php');
+        file_exists($cacheConfig) ? require_once app()->configPath('wordpress.php') : null;
```
Comment (jlambe) [inline src/Core/Bootstrap/ConfigurationLoader.php:89]: As we don't return anything here, I suggest to update the method body like so:
```suggestion
        if (! file_exists($cacheConfig)) {
            return;
        }
        
        require_once app()->configPath('wordpress.php');
```

File: src/Core/Bootstrap/ConfigurationLoader.php [hunk 6]
```diff
@@ -79,6 +74,30 @@ protected function loadConfigurationFiles(Application $app, RepositoryContract $
...
+    }
+
+    /**
+     * Check if the WordPress config constants are already defined.
+     * @return bool
```
Comment (jlambe) [inline src/Core/Bootstrap/ConfigurationLoader.php:94]: Remove docblock return statement as it is defined in the method signature.
```suggestion
```

File: src/Core/Bootstrap/ConfigurationLoader.php [hunk 7]
```diff
@@ -79,6 +74,30 @@ protected function loadConfigurationFiles(Application $app, RepositoryContract $
         }
     }
 
+    /**
+     * Get the path to the routes cache file.
+     *
+     * @return string
+     */
+    protected function maybeForceWpConfigInclude()
```
Comment (jlambe) [inline src/Core/Bootstrap/ConfigurationLoader.php:82]:
```suggestion
    protected function maybeForceWpConfigInclude(): void
```

**Review by jlambe — CHANGES_REQUESTED** [review]
@ogorzalka Last bits of change 😃 then I think we're good.

**Comment by ogorzalka** [comment]
I think it's ok @jlambe :)

**Review by jlambe — APPROVED** [review]
@ogorzalka 👍

**Pull request merged**